Documents/filing-status.md

```diff
@@ -0,0 +1,25 @@
+## Filing status
+
+Every filing will store a `FilingStatus` object with the following structure:
+
+* `code`: `Integer`
+* `message`: `String`
+
+For example:
+
+```json
+{
+  "id": 3,
+  "status": {
+    "code": 1,
+    "message": "not-started"
+  }
+}
+```
+
+In order to track the status of a filing for a financial institution, the following states are captured by the backend:
+
+* `1`: `NotStarted` - A filing has been opened but no submission has been submitted for the filing.
```

Comment:
Should this be `not-started` to match the message above? Probably the same for the other statuses too.